src/lib/LUA/tx_devLUA.cpp

```diff
@@ -101,6 +102,15 @@ static struct luaItem_selection luaSwitch = {
   };
 #endif
 
+#if defined(GPIO_PIN_NSS_2)
```

Comment:
This define isnt required and only allows Gemini tx modules to select Mavlink... unless thats what you want 😃

Reply:
Perplexingly, I have two TXes flashed with this (ES24TX Slim Pro & ES900TX) that aren't Gemini and can absolutely select the MAVLink Link Mode.

Reply:
Ah, it's probably because it's not GPIO_PIN_NSS_2 != UNDEF_PIN.

Weird and I'll have to go back and check the other Antenna mode option.  This shouldn't be available on single radio TX modules. 

Either way, there's some clean up to do here.

Reply:
Because the define just calls a function so it can be determined at runtime. 
See https://github.com/ExpressLRS/ExpressLRS/blob/master/src/include/target/Unified_ESP32_TX.h#L53